packages/cosmos_ui_components/lib/cosmos_text_theme.dart

```diff
@@ -63,5 +63,11 @@ abstract class CosmosTextTheme {
         height: 1.23,
       );
 
+  static TextStyle get labelM => const TextStyle(
```

Comment:
we take the typography namings straight from figma, as far as I can see, the wallet name uses `title0Medium`
![CleanShot 2021-11-30 at 17 51 52@2x](https://user-images.githubusercontent.com/1711474/144091561-21722b92-37a5-4c14-97bb-bc22b62ab7c1.png)

Reply:
Ah, that makes sense. In one of the places in figma I found some `label/X` styles to be marked as deprecated, so lets use title0Medium then if possible